rpc/core/tx.go

```diff
@@ -20,7 +18,7 @@ import (
 func (env *Environment) Tx(_ *rpctypes.Context, hash []byte, prove bool) (*ctypes.ResultTx, error) {
 	// if index is disabled, return error
 	if _, ok := env.TxIndexer.(*null.TxIndex); ok {
-		return nil, fmt.Errorf("transaction indexing is disabled")
+		return nil, ErrTxIndexing
```

Comment:
Name the const `ErrTxIndexingDisabled` for clarity?

Reply:
Done.